flopy/mf6/utils/model_splitter.py

```diff
@@ -2893,12 +2893,43 @@ def _remap_package(self, package, ismvr=False):
                 elif isinstance(value, mfdataarray.MFArray):
                     mapped_data = self._remap_array(item, value, mapped_data)
 
-                elif isinstance(value, mfdatalist.MFTransientList):
+                elif isinstance(
+                    value, mfdatalist.MFTransientList
+                ) or isinstance(value, mfdataplist.MFPandasTransientList):

```

Comment:
The isinstance statement can be simplified `isinstance(value, (mfdatalist.MFTransientList, mfdataplist.MFPandasTransientList))

Reply:
Simplified